src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    onsubmit="onSubmit()">
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input id="surname"
+          type="text"
+          name="surname"
+          autocomplete="off">
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input id="name"
+          type="text"
+          name="name"
+          autocomplete="off">
+        </div>
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input id="age"
+          type="number"
+          name="age"
+          value="12"
+          min="7"
+          max="100">
+        </div>
+        <div class="form-field">
+          <label for="birth">Full date of birth:</label>
+          <input id="birth"
+          type="date"
+          name="date of birth">
```

Comment:
Use camelCase for name attribute. It should not include spaces.
```suggestion
          name="dateOfBirth">
```